lenovo/thinkpad/x250.nix

```diff
@@ -0,0 +1,5 @@
+{ config, pkgs, ... }:
+
+{
+  imports = [ ./general-intel.nix ];
+}
```

Comment:
I'd vote for removing `x250.nix` as all of its values are general defaults.
Only if a model requires special customizations, a special file should be added, right?

Reply:
👍